src/components/sidebar/MenuMap.js

```diff
@@ -43,26 +43,28 @@ class MenuMap extends Component {
         </span>
         { !/Benchmarks/.test(item.title) &&
         <div className="sidebar--menu--detail--button--container">
-          <span
+          <FileText
             className={`sidebar--menu--detail--button solution ${this.props.theme} cm-variable`}
             id={'SOLUTION__' + id}
-            onClick={this.selectSolution}>
-            Solution
-          </span>
-          <span
+            data-tip
+            data-for='solutionTip'
+            onClick={this.selectSolution} />
+          <ReactTooltip id='solutionTip' type='dark' effect='solid' delayShow={300}>Solution</ReactTooltip>
```

Comment:
Wonder if there's a way to style these? do they accept a `className` prop or `style` object that gets passed down to the underlying `div` or whatever it is under the hood I wonder? I'll have to check out the docs. Might be cool to style these with the themes, or too much you think? The white on black contrast is def really nice.